src/pages/marketplace/slideshow.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Slide } from 'react-slideshow-image';
+import 'react-slideshow-image/dist/styles.css'
+import Link from "@docusaurus/Link";
+
+
+const slideImages = [
+  {
+    url: 'https://raw.githubusercontent.com/demisto/content-docs/148c5a9e7153b1688acb2f331b414e30a3f75f8c/src/pages/marketplace/images/Marketplace_Vertical_1.png',
```

Comment:
Link to the images with relative links and not via the `raw.github..` link. The images should go into the `static/img` dir. You can open there a subdir and put all related images to the sliideshow.